control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -1320,6 +1320,87 @@ paths:
           $ref: '#/components/responses/ServerError'
       security:
         - JWT: []
+  '/pools/{pool_id}/label/{key}={value}':
```

Comment:
Ah got it it's a single value here but the grpc accepts a map, so in the future we can pass a list of labels through body json if we want to